### ISSUE TITLE
Great Idea to Open Create This Repo!

### ISSUE THREAD

**Issue by gbrunner** [gchadala/Final-Prj-Timeline#1]
@gchadala Great idea to share your project here! Let's see you share your code here. 👍

**Comment by gchadala** [comment]
Hi Professor Brunner,

 Would showing this code on GitHub be advisable for presenting the project?
I would think it would be awkward to use a PowerPoint for this course. I
wanted to know, how do you import a code into a repository? I had a hard
time following when we did this last month.

Govind

**Comment by gbrunner** [comment]
@gchadala Yes. You can do that. I'd advise you to fill out the wiki page too then.

To import the code to the repo, do you have SourceTree installed?

**Comment by gchadala** [comment]
I do have SourceTree, but I was not sure where to go once I had a main
'Branch'. Somehow I accidentally deleted al my work, and just redone my
progress just now. Could you explain the steps to importing the code?

**Comment by gbrunner** [comment]
@gchadala 
1. Clone the repo to your computer:
![image](https://cloud.githubusercontent.com/assets/5703643/25499864/87b322be-2b52-11e7-9881-6a0e9f1315b8.png)

2. add your code to the local folder that you cloned the repo to.
![image](https://cloud.githubusercontent.com/assets/5703643/25499939/cca5d272-2b52-11e7-8868-ce5b4b7667c6.png)

3. Go back to source tree.
![image](https://cloud.githubusercontent.com/assets/5703643/25499960/dfb1bfca-2b52-11e7-95ac-95e185acbd19.png)

4. click "stage all"
![image](https://cloud.githubusercontent.com/assets/5703643/25499981/f109eef0-2b52-11e7-9305-a256cf97a6d8.png)

5. Add some comments, then "commit"

6. Then go to the "Push" button and "Push"

**Comment by gchadala** [comment]
Okay, the code is now present on GitHub. I think I'll be able to add in
one, maybe two more maps for the timeline, would that be enough for the
presentation?

**Comment by gbrunner** [comment]
I think so. Don't forget to:
- update your code on github if you change it
- show some nice maps in the presentation.

You can take screenshots and drag and drop screenshots of your maps into the wiki on your github page.
